ansible/roles/hkcam/files/update_hkcam.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+CURRENT="$(/usr/bin/hkcam -version)"
+LATEST="$(curl -s https://api.github.com/repos/brutella/hkcam/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")')"
+
+if [ $CURRENT != ${LATEST#?} ]; then
```

Comment:
We may want to consider the case where  *$CURRENT* is higher than *$LATEST*, eg during development.

Reply:
Thank you for your feedback. Maybe you can add a tag with 'beta' or something like that, which we can also compare too? What is your suggestion based on your workflow? 

> Please use `FirmwareRevision` also in the `acceessory.Info` struct.
I updated it!